nautobot/ipam/tests/test_views.py

```diff
@@ -94,6 +103,8 @@ def setUpTestData(cls):
             "namespace": prefixes[0].namespace.id,
             "add_prefixes": [prefixes[0].id],
             "remove_prefixes": [prefixes[1].id],
+            "add_virtual_device_contexts": [vdcs[0].id, vdcs[1].id],
```

Comment:
Shouldn't we also be augmenting the `form_data` with a VDC as well for this test?